test/create_examples_emr_test.go

```diff
@@ -0,0 +1,137 @@
+package test
+
+import (
+	"fmt"
+	"strings"
+	"testing"
+
+	"github.com/gruntwork-io/terratest/modules/aws"
+	"github.com/gruntwork-io/terratest/modules/random"
+	"github.com/gruntwork-io/terratest/modules/terraform"
+	test_structure "github.com/gruntwork-io/terratest/modules/test-structure"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+func initTestCases() []EmrTestCase {
+	return []EmrTestCase{
+		{
+			testName: "Hbase_Test_1",
+			vars: map[string]interface{}{
```

Comment:
Is there a particular reason for not reusing the config maps?

Reply:
they have few differences, vpc_cidr and tfDir... also this way it's easy to have more disparity between tests if needed.